packages/playwright-test/src/util.ts

```diff
@@ -305,3 +305,12 @@ export function envWithoutExperimentalLoaderOptions(): NodeJS.ProcessEnv {
     result.NODE_OPTIONS = result.NODE_OPTIONS.replace(substring, '').trim() || undefined;
   return result;
 }
+
+export function determinePackageManager() {
+  if (process.env.npm_config_user_agent) {
+    if (process.env.npm_config_user_agent.includes('yarn')) return 'yarn';
```

Comment:
We can use process.env.npm_config_user_agent? onstruct for brevity. Also I don't think `pnpm playwright` does the right thing, you probably want `pnpm exec playwright`.

Reply:
I believe the right way to get package manage of use is this.
I changed the code to use `getFromENV` like another files of project
And change the message from `pnpm` to `pnpm exec`

If you have some suggest or believe this is not a better way to do that, let me know